### PR TITLE
Flatpak manifest updates from the Flathub's PR

### PR DESCRIPTION
**Pull request by besdar**
Hello @rustdesk!

As you know, I recently [passed a code review in the Flathub repository](https://github.com/flathub/flathub/pull/5670). During the review, Flathub's reviewers suggested a few changes to the manifest file and asked that you also add a metainfo file to your repository. I have checked locally that everything seems to work with updated manifes, so I think it would be a good thing to modify your manifest file as well, to ensure consistency between your flatpak configuration and the one from the Flathub. If you decide to merge these changes, the only difference between the configurations would be the `"--talk-name=org.freedesktop.Flatpak"` finish argument, which was removed in Fluthub's version due to security issues.

Please feel free to implement only the changes that you find suitable for your Flatpak build. Thank you for considering this suggestion.

### CODE REVIEW

**Comment by rustdesk** [comment]
Thanks

File: flatpak/rustdesk.json [hunk 1]
```diff
@@ -26,32 +43,25 @@
       "name": "rustdesk",
       "buildsystem": "simple",
       "build-commands": [
-        "bsdtar -zxvf rustdesk.deb",
-        "tar -xvf ./data.tar.xz",
-        "cp -r ./usr/*  /app/",
+        "bsdtar -Oxf rustdesk.deb data.tar.xz | bsdtar -xf -",
+        "cp -r usr/* /app/",
         "mkdir -p /app/bin && ln -s /app/lib/rustdesk/rustdesk /app/bin/rustdesk",
-        "mv /app/share/applications/rustdesk.desktop /app/share/applications/com.rustdesk.RustDesk.desktop",
```
Comment (rustdesk) [inline flatpak/rustdesk.json:33]: Without this change, where is the desktop icon? Or 24.08 has changed the rule?

Comment (besdar) [reply flatpak/rustdesk.json:33]: @rustdesk in the `.deb` package. 

ref: 
1) https://github.com/flathub/flathub/pull/5670#discussion_r1788043449
2) https://github.com/flathub/flathub/pull/5670#discussion_r1788042435
3) https://github.com/flathub/flathub/pull/5670#discussion_r1788039622

Comment (besdar) [reply flatpak/rustdesk.json:33]: As far as I know, 24.08 doesn't have any changes for icons. The flatpak builder just takes the image for the icon and `.desktop` from the `.deb` package. It works on 23.08 too.

**Comment by rustdesk** [comment]
CI failed https://github.com/rustdesk/rustdesk/actions/runs/11214038944/job/31175489848

**Comment by rustdesk** [comment]
I would revert this first.

**Pull request merged**